app/routers/guest/applications.py

```diff
@@ -717,3 +718,41 @@ async def find_alternative_credit_option(
             borrower=new_application.borrower,
             award=new_application.award,
         )
+
+
+@router.get(
+    "/applications/uuid/{uuid}/access-external-onboarding",
+    tags=[util.Tags.applications],
+)
+async def access_external_onboarding(
+    session: Session = Depends(get_db),
+    application: models.Application = Depends(
+        dependencies.get_scoped_application_as_guest_via_uuid(statuses=(models.ApplicationStatus.SUBMITTED,))
+    ),
+) -> RedirectResponse:
+    """
+    Set the application borrower_accessed_external_onboarding_at and redirects to
+    lender.external_onboarding_system_url.
+
+    :param session
+    :param application
```

Comment:
The consultants had lots of docs/comments that just said exactly what the code did. We don't need that kind. In this case the return docs are enough.

```suggestion
```